keps/sig-node/20190717-seccomp-ga.md

```diff
@@ -262,6 +318,28 @@ type SeccompProfileSet struct {
 
 ## Design Details
 
+
+### Failure and Fallback Strategy
+
+There are different scenarios in which applying seccomp may fail, below are the ones we mapped 
+and their outcome once this KEP is implemented:
+
+| Scenario | API Server Result | Kubelet Result |
+|--------------------------------------------------------------|------------------------------|----------------------------------------------------------------------------------------------|
+| 1) Using custom or `runtime/default` profile when container runtime does not support seccomp. | Pod created | The outcome is container runtime dependent. In this scenario containers may 1) fail to start or 2) run normally without having its policies enforced. |
+| 2) Using custom or `runtime/default` profile that restricts syscalls a container is trying to make. | Pod created | The outcome is workload and seccomp dependent. In this scenario containers may 1) fail to start, 2) misbehave or 3) log violations. |
```

Comment:
which log would the violations appear in (container, kubelet, cri, other...)?

Reply:
Seccomp is enforced at kernel level, so the violations will end-up on the kernel logs. I will amend the text to clarify that.